ml-agents-envs/mlagents_envs/side_channel/engine_configuration_channel.py

```diff
@@ -25,8 +27,16 @@ class EngineConfigurationChannel(SideChannel):
      - int qualityLevel;
      - float timeScale;
      - int targetFrameRate;
+     - int captureFrameRate;
     """
 
+    class ConfigurationType(IntEnum):
+        SCREEN = (0,)
```

Comment:
no commas (these are 1-tuples, not ints)

Reply:
(still the same formatting bug I am seeing) 
Will change and be more careful